### PR TITLE
Added links and Prague specific information in a friendly tone of voice.

### PR DESCRIPTION
**Pull request by ntoll**

### CODE REVIEW

File: data/pages-content/where.md [hunk 1]
```diff
@@ -4,34 +4,47 @@ subtitle: EuroPython 2023 will be held in Prague!
...
-Prague is one of the top five city destinations in Europe. The beautiful historic 
-centre is a UNESCO world heritage site. So, in addition to the conference, 
-there's a LOT to see, do and enjoy. Prague has an extensive and modern public 
-transport system, so travelling around won't be a problem and getting there is 
-easy via well serviced train, air and road infrastructure. With an average 
-July high of 25°c, remember to pack your shorts and sandals with your laptop, 
-along with a spirit of adventure to explore one of the most visited and vibrant 
-cities in Europe.
+## Prague
+<img src="/img/Prague.jpeg" width="3279" height="1785"/>
```
Comment (ntoll) [inline data/pages-content/where.md:10]: Will the width and height work on mobile..?

File: data/pages-content/where.md [hunk 2]
```diff
@@ -4,34 +4,47 @@ subtitle: EuroPython 2023 will be held in Prague!
...
 
-As per EuroPython tradition, we shall walk around, chat, laugh and explore the
-city together. If you are not the “walking-type”, 
-[Prague's public transport website](https://prague.org/public-transportation-in-prague/)
-will give you all the information you need.
+<br></br>
 
-### The Venue
+The capital of the Czech Republic and sitting on the [Vltava River](https://www.youtube.com/watch?v=l6kqu2mk-Kw),the beautiful historic centre is a UNESCO world heritage site. There's a LOT to see, do and enjoy, in addition to the conference! The ancient fortress [Vyšehrad](https://www.vysehradtickets.com/vysehrad/) is just 10-min walk away from the conference venue.
```
Comment (ntoll) [inline data/pages-content/where.md:14]: Prague is the capital.... etc....

File: data/pages-content/where.md (hunk 1 above)
Comment (hypha) [reply data/pages-content/where.md]: it does but I didn't need to put it there. So deleted!

File: data/pages-content/where.md (hunk 2 above)
Comment (hypha) [reply data/pages-content/where.md]: fixed in latest https://github.com/EuroPython/website/pull/228/commits/f8821e02c177bcec60dce019ab78a4b6b5502257, I hope.

**Review by hypha — APPROVED** [review]

**Pull request merged**